src/ci/github-actions/jobs.yml

```diff
@@ -294,7 +294,7 @@ auto:
 
   - image: x86_64-apple-2
     env:
-      SCRIPT: ./x.py --stage 2 test tests/ui tests/rustdoc tests/run-make-fulldeps
+      SCRIPT: ./x.py --stage 2 test tests/ui tests/rustdoc
```

Comment:
Shouldn't we run `run-make` tests here?

cc @Kobzol

Reply:
We should be running them [here](https://github.com/rust-lang/rust/pull/126155/files#diff-ef98dea3810f7940c389a6d469a0fad6c21da049c0ad16e9b0f27768c87dce2cR40), since that runs all tests except for ui and rustdoc.